cli/src/main/scala/org/apache/celeborn/cli/config/CliConfigManager.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.cli.config
+
+import java.io.{File, FileInputStream, FileOutputStream}
+import java.time.LocalDateTime
+import java.util.Properties
+
+import org.apache.celeborn.cli.common.CliLogging
+import org.apache.celeborn.common.util.Utils
+
+case class CliConfig(cliConfigData: Map[String, String])
+
+object CliConfigManager {
+  val cliConfigFilePath: String =
+    s"${sys.env.getOrElse("EXAMPLE_CONF_DIR", sys.env("HOME"))}/celeborn-cli.conf"
```

Comment:
What is `EXAMPLE_CONF_DIR` here ?

Reply:
Also, make it `.celeborn-cli.conf` is we are writing to home directory

Reply:
`EXAMPLE_CONF_DIR` --  did not properly stash my local changes :)

Reply:
updated this now.